### DISCUSSION TITLE
Formula issue [Oct 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4260] [Discord Threads]
> 🙋‍♂️ **DtNeo**  
#### What's wrong ? database ?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1031654349002248213/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1031654349367160943/unknown.png)  
![image2](https://cdn.discordapp.com/attachments/661905455894888493/1031654349832728616/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **DtNeo**  
#### I did a container with postgres, I can divide. I will try to understand what's wrong with mysql  
#  
> 🙋‍♂️ **DtNeo**  
#### I don't have enought log. no error.   
#### I will continue on postgres, but I don't like it  
#  
> 🙋‍♂️ **wkw**  
####   
![image0](https://cdn.discordapp.com/attachments/1031654349971132540/1031783902026666084/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### if u edit the formula again, do u see the same formula or it s broken?  
#  
> 🙋‍♂️ **wkw**  
#### here's my project info   
####    
#### Node: **v16.14.0**   
#### Arch: **arm64**   
#### Platform: **darwin**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.1**  
#  
> 👨‍💻 **pranavxc**  
#### <@442352221137403904> : can you share the docker logs as well ?  
#  
> 🙋‍♂️ **DtNeo**  
#### docker image : nocodb/nocodb:latest + mysql:5.7   
#### Arch amd64  
#  
> 🙋‍♂️ **DtNeo**  
#### what log do you need ?  
#  
> 🙋‍♂️ **DtNeo**  
#### i'm in the docker. noting in /var/log  
#  
> 🙋‍♂️ **DtNeo**  
#### I see the formula as I wrote it. if I change for 1+1, it work well and I see all line reappear  
#  
> 👨‍💻 **pranavxc**  
#### you can do `docker logs -f container_name`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1031654349971132540)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
